.travis.yml

```diff
@@ -26,10 +25,18 @@ addons:
       - google-chrome-beta
 
 before_install:
-  # Get firefox beta and extract to tar for the test runner to use.
-  - wget 'https://download.mozilla.org/?product=firefox-beta-latest&lang=en-US&os=linux64' -O firefox-beta.tar.bz2 && tar xvjf firefox-beta.tar.bz2
+  # Get firefox beta and nightly and extract to tar for the test runner to use.
+  - ./project/download/firefox-beta.sh
+  - ./project/download/firefox-nightly.sh
   # Get marionette executable
-  - if [[ "$TRAVIS_OS_NAME" = "linux" ]]; then ./project/get-marionette.sh; fi
+  - if [[ "$TRAVIS_OS_NAME" = "linux" ]]; then ./project/download/firefox-beta.sh; fi
```

Comment:
Could all these lines go in a single script? Just seems tidier, given they have the same condition.

Also, it looks like you run `./project/download/firefox-nightly.sh` twice on Linux - on lines 30 and 33